### ISSUE TITLE
drush entity:save fails with chunks option

### ISSUE THREAD

**Issue by drubb** [drush-ops/drush#6000]
Drush Version: 12.5.2

Currently, using the --chunks option for entity:save is not possible in Drush 12, it throws a type error.

Example: `drush esav node --chunks=100`

```
[error]  TypeError: array_chunk(): Argument #2 ($length) must be of type int, string given in array_chunk() (line 118 of /app/vendor/drush/drush/src/Commands/core/EntityCommands.php) #0 /app/vendor/drush/drush/src/Commands/core/EntityCommands.php(118): array_chunk(Array, '100', true)
```

As far as I can see this has been fixed in 13.x, but maybe it still needs to be backported to 12.x

**Comment by weitzman** [comment]
Fixed in #6001

**Comment by Chi-teck** [comment]
Can we have a patch release for this fix?
